summary36.txt

```diff
@@ -0,0 +1,100 @@
+Interface Lava stairs message
+Bugfixes Display proper amount of byproducts in crafting GUI
+Bugfixes Vehicle wheels now will be damaged on lava
+Infrastructure [CR] Various optimizations reported by static code analysis
```

Comment:
Would you please remove tags like [CR] and [RDY]?

Reply:
I removed all of those, and also submitted a new PR to do the same for summaries 37 and 38.